spec/filters/jdbc/lookup_spec.rb

```diff
@@ -124,6 +145,41 @@ module LogStash module Filters module Jdbc
         expect(event.get("server")).to eq(records)
       end
     end
+
+    describe "lookup operations with prepared statement" do
+      let(:local_db) { double("local_db") }
+      let(:lookup_hash) do
+        {
+          "query" => "select * from servers WHERE ip LIKE ?",
```

Comment:
Can we have a test with >1 parameters?

Reply:
Done, thanks for suggestion, because spotted a test that didn'tt do its job:-)